bundles/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@brave/sync-client",
+  "version": "0.1.0",
```

Comment:
i don't remember if there are any clients currently using this NPM package (as opposed to downloading sync bundle from github releases). if so i can publish an update.